test/integration/targets/module_utils/library/test_tmpdir.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
```

Comment:
This test belongs somewhere else.  This target is for testing that pluginloader loads module_utils correctly.